### ISSUE TITLE
add to HACS

### ISSUE THREAD

**Issue by andordavoti** [postlund/home-card#22]
Could you add this card to HACS to simplify install and updates?
Thanks for the hard work btw:))

**Comment by postlund** [comment]
Yeah, that is my intention. It requires some additional work on my end because of all the images. My focus is currently on other things, but it is definitely in the backlog!

**Comment by andordavoti** [comment]
Thanks!

**Comment by gdreelin** [comment]
It works by putting in HACS as a custom repository and allows you to update.
